saskatoon/harvest/views.py

```diff
@@ -217,7 +246,14 @@ def get_success_message(self, cleaned_data) -> str:
                 self.object.save()
                 return ""
 
-            person = self.object.pick_leader.person
+            maybe_auth = self.object.pick_leader
+
+            assert maybe_auth is not None
+
+            person = maybe_auth.person
+
+            assert person is not None
+
```

Comment:
Sorry I missed this, see https://github.com/LesFruitsDefendus/saskatoon-ng/pull/608#discussion_r2442672065